projects/storefrontapp/src/app/spartacus/features/b2b-digital-payments/checkout-b2b-dp-feature.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CHECKOUT_FEATURE } from '@spartacus/checkout/base/root';
+import { CmsConfig, I18nConfig, provideConfig } from '@spartacus/core';
+import {
+  dpTranslationChunksConfig,
+  dpTranslations,
+} from '@spartacus/digital-payments/assets';
+@NgModule({
+  providers: [
+    provideConfig(<CmsConfig>{
+      featureModules: {
+        [CHECKOUT_FEATURE]: {
+          module: () =>
+            import('./b2b-digital-payments.module').then(
+              (m) => m.B2BDigitalPaymentsModule
+            ),
+        },
+      },
+    }),
+    provideConfig(<I18nConfig>{
+      i18n: {
+        resources: dpTranslations,
+        chunks: dpTranslationChunksConfig,
+        fallbackLang: 'en',
+      },
+    }),
+  ],
+})
+export class CheckoutB2bDpFeatureModule {}
```

Comment:
Didn't we agree to have a "wrapper" module only for _one_ checkout? Either the base, b2b _or_ repl?

I think it's enough if just keep the `projects/storefrontapp/src/app/spartacus/features/digital-payments-feature.module.ts`